amm_info updates (#1704)
XRPLF/xrpl-dev-portal: content/references/http-websocket-apis/public-api-methods/account-methods/account_objects.md

| `Field`                  | Type             | Description                    |
|:-------------------------|:-----------------|:-------------------------------|
| `account`                | String           | A unique identifier for the account, most commonly the account's address. |

> | `type`                   | String           | _(Optional)_ If included, filter results to include only this type of ledger object. The valid types are: `check`, `deposit_preauth`, `escrow`, `nft_offer`, `offer`, `payment_channel`, `signer_list`, `state` (trust line), and `ticket`. <!-- Author's note: Omitted types that can't be owned by an account --> |

Reviewer: ```suggestion
| `type`                   | String           | _(Optional)_ Filter results by a ledger object type. The valid types are: `check`, `deposit_preauth`, `escrow`, `nft_offer`, `offer`, `payment_channel`, `signer_list`, `state` (trust line), and `ticket`. <!-- Author's note: Omitted types that can't be owned by an account --> |
```
